### ISSUE TITLE
Add ShellCheck's "source.fixAll.shellcheck" functionality

### ISSUE THREAD

**Issue by j-lakeman** [bash-lsp/bash-language-server#1040]
### What is the problem this feature will solve?

lots of little bash problems at once e.g. forgotten quotation marks

### What is the feature you are proposing to solve the problem?

automatic fixing of simple issues on save e.g. inserting quotation marks

### What alternatives have you considered?

Using ShellCheck alongside Bash IDE, but only for this functionality, as Bash IDE already has all of ShellCheck's other features. This would implement one more quite handy little feature IMHO.

**Comment by chrisgrieser** [comment]
I agree. To truly make bash-lsp standalone, it would also require bundling shellcheck though (#663)

**Comment by skovhus** [comment]
Makes sense. Contributions are more than welcome.
